string waypoints_yaml_file
---
bool success
string status_msg
